.env

```diff
@@ -0,0 +1,14 @@
+RESMED_LOGIN=resmed_user_email
```

Comment:
we've "lost" all the documentation of the parameters. Maybe add a comment that points to the template config file?